support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -47,12 +41,12 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     services: Services
   ): FutureHandlerResult = {
 
-    val createSubscription = new ZuoraSubscriptionCreator(
+    val createSubscription = ZuoraSubscriptionCreator.construct(
```

Comment:
Could use apply here? I know we're avoiding apply but in my mind that's for anything other than constructors